### ISSUE TITLE
test transfer learning algorithm on image classification application

### ISSUE THREAD

**Issue by mkh48v** [mkh48v/omnis#223]
check transfer learning performance when a user changes only a few classes.

first, train a model with a caltech101 dataset.

secondly, changing or adding a few classes to a train dataset(ex. add new classes from caltech256).

then, use a pre-trained model and apply transfer learning.

finally, check accuracy of the model with data of new class and data of original class.

if transfer learning works well, i think we can change image_classification application like below.

def train(use_transfer_learning = True)

if transfer_learning flag is true, train method will apply a transfer learning if class_indices changed

**Comment by mkh48v** [comment]
check

https://keras.io/examples/mnist_transfer_cnn/

https://www.google.com/search?q=keras+transfer+learning&oq=keras+transfer+learning&aqs=chrome..69i57j0l5.4670j0j4&sourceid=chrome&ie=UTF-8

**Comment by mkh48v** [comment]
i'm not sure this works on models we use

if we can't use transfer learning. i think we should investigate methods to handle a small change of classes

**Comment by 2013-11390** [comment]
It's impossible to use transfer learning with changing classes number if we use keras application layers.
